src/docs/implementations/dynatrace.adoc

```diff
@@ -89,15 +85,17 @@ MeterRegistry registry = new DynatraceMeterRegistry(dynatraceConfig, Clock.SYSTE
 
 These properties can also be set via the Spring Boot properties file.
 It is possible to reference environment variables using the Spring property placeholder notation here (`${VAR_NAME}`).
+When running the Micrometer exporter for Spring Boot, `v2` is used as the default API version.
 
 [source,yml]
 ----
 management.metrics.export.dynatrace:
-    api-version: v2
+    # for SaaS: https://{your-environment-id}.live.dynatrace.com/api/v2/metrics/ingest
+    # for managed deployments: https://{your-domain}/e/{your-environment-id}/api/v2/metrics/ingest
+    uri: ${DT_METRICS_INGEST_URL}
 
-    # e.g.: https://{your-environment-id}.live.dynatrace.com/api/v2/metrics/ingest
-    uri: ${ENV_VAR_DT_INGEST_URI}
-    api-token: ${ENV_VAR_DT_INGEST_TOKEN}
+    # should not be hardcoded but rather read from a secure source.
```

Comment:
```suggestion
    # should not be hardcoded but rather read from a secure source like env vars
```
Since that's what we're already doing in this example.